.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  workflow_dispatch:
```

This will do automated preReleases for the `next` branch. IMHO its handy but you can decide if you want to use it or not.

```suggestion
  workflow_dispatch:
  push:
    branches:
      - next
```